sample-code/src/orchestration.ts

```diff
@@ -744,11 +744,22 @@ export async function orchestrationTranslation(): Promise<OrchestrationResponse>
     translation: {
       input: buildTranslationConfig('input', {
         sourceLanguage: 'en-US',
-        targetLanguage: 'de-DE'
+        targetLanguage: 'de-DE',
+        translateMessagesHistory: true,
+        applyTo: [
+          {
+            category: 'placeholders',
+            items: ['user_input'],
+            sourceLanguage: 'en-US'
+          }
+        ]
       }),
       output: buildTranslationConfig('output', {
         sourceLanguage: 'de-DE',
-        targetLanguage: 'fr-FR'
+        targetLanguage: {
+          category: 'placeholders',
+          items: ['assistant_response']
+        }
```

Comment:
[req] I tested the sample code and in it's current state it does not work. The following changes had to be made to make it work:

1.  `sourceLanguage` had to be removed from input config, as `applyTo` cannot be combined with a `sourceLanguage` property at the root.
2. items in output config cannot contain `assistant_response` as it is not a placeholder value. Change it to `user_input` instead.